src/Client.cpp

```diff
@@ -33,6 +33,7 @@
 #include "PacketTimes.h"
 #include "Switches.h"
 
+
```

Comment:
No actually, I was including another header for testing and forget to clean the new line after removing the included header/